ksim/randomization.py

```diff
@@ -251,3 +253,45 @@ def __call__(self, model: PhysicsModel, rng: PRNGKeyArray) -> dict[str, Array]:
         )
         new_qpos = jnp.concatenate([model.qpos0[:7], new_qpos])
         return {"qpos0": new_qpos}
+
+
+@attrs.define(frozen=True, kw_only=True)
+class IMUAlignmentRandomizer(PhysicsRandomizer):
+    site_name: str = "imu_site"
+    tilt_std_rad: float = 0.0349066  # 1σ for roll & pitch (2 deg in rad)
+    yaw_std_rad: float | None = None  # 1σ for yaw if desired
+    translate_std_m: float | None = None  # 1σ in metres
+
+    def __call__(self, model: PhysicsModel, rng: PRNGKeyArray) -> dict[str, Array]:
+        # sample small rotations from normal distribution
+        rng, sub = jax.random.split(rng)
+        rx, ry = jax.random.normal(sub, (2,)) * self.tilt_std_rad
+
+        rz = jnp.array(0.0)
+        if self.yaw_std_rad is not None:
+            rng, sub = jax.random.split(rng)
+            rz = jax.random.normal(sub) * self.yaw_std_rad
+
+        # small angle approximation: q ≈ [1, θx/2, θy/2, θz/2]
```

Comment:
nit: [Google-style python comments](https://google.github.io/styleguide/pyguide.html)

i.e., capitalize + use period at the end